umbraco/Umbraco.CMS.Backoffice
# Bugfix/document blueprint entity actions

## Description

Fixes the available entity actions for document blueprints.

* Root -> Create, Reload
* Entity -> Delete, Move To
* Folder -> Create, Rename, Delete, Reload

## How to test
* Make some random tests by creating, deleting etc. in the document blueprint tree and make sure it behaves as expected.

## Types of changes




## Motivation and context



## How to test?

## Screenshots (if appropriate)

## Checklist
